library/components/tableComponents/SnykPager.vue

```diff
@@ -2,60 +2,56 @@
   <div>
     <div
       v-if="numberOfItems && numberOfItems > itemsPerPage"
-      class="pagination-footer flex justify-center relative h-8"
+      class="pagination-footer flex justify-between p-[12px] items-center"
```

Comment:
I'd like to elevate this to the table level, to keep the padding in sync between these components. I want to keep code movement to a minimum in this PR.